.github/workflows/build_linux_app.yml

```diff
@@ -0,0 +1,21 @@
+name: Build_Linux_app
+
+on:
+  push:
+    tags:
+    - "*"
+
+jobs:
+  Build_Linux_app:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+    - run: |
+        sudo apt-get install libsdl2-2.0-0
+        python -m pip install pyinstaller
+        python setup.py clean bundle
```

Comment:
Maybe split those 3 steps up into separate ones? It's faster to spot where things went wrong when the output is grouped separately.